subsys/net/lib/wifi_ready/Kconfig

```diff
@@ -0,0 +1,27 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+menuconfig WIFI_READY_EVENT_HANDLING
```

Comment:
I don't see a `endmenu`, is that not required?

Reply:
It is required for `menu` option, not for `menuconfig`.